include/meson.build

```diff
@@ -0,0 +1,15 @@
+###########
+# Headers #
+###########
+
+if not meson.is_subproject()
+  install_headers(
+    files([
+      'spoa/alignment_engine.hpp',
+      'spoa/graph.hpp',
+      'spoa/spoa.hpp',
+    ]),
+    subdir : 'spoa')
+endif
```

Comment:
I know this is the PB pattern, but let's change it:
1. put `spoa_include_directories = include_directories(['.'])` first
2. [then](https://mesonbuild.com/Reference-manual_functions.html#subdir_done)
```
if meson.is_subproject()
  subdir_done()
endif
```
3. `install_headers` unconditionally